### ISSUE TITLE
The supabase url should not contain (auth/v1), supabase-kt handles the url endpoints. 

### ISSUE THREAD

**Issue by yevgnenll** [tschuehly/htmx-supabase-spring-boot-starter#6]
hi tschuehly

i'm new user supabase with spring boot 3.2.x
i've got error below text when spring boot start

```
Caused by: java.lang.IllegalStateException: The supabase url should not contain (auth/v1), supabase-kt handles the url endpoints. If you want to use a custom url for a module, specify it within their builder but that's not necessary for normal supabase projects
	at io.github.jan.supabase.SupabaseClientBuilder.<init>(SupabaseClientBuilder.kt:69) ~[supabase-kt-jvm-2.0.0.jar:na]
	at de.tschuehly.htmx.spring.supabase.auth.SupabaseAutoConfiguration.supabaseClient(SupabaseAutoConfiguration.kt:77) ~[htmx-supabase-spring-boot-starter-0.3.0.jar:na]
	at de.tschuehly.htmx.spring.supabase.auth.SupabaseAutoConfiguration$$SpringCGLIB$$0.CGLIB$supabaseClient$1(<generated>) ~[htmx-supabase-spring-boot-starter-0.3.0.jar:na]
	at de.tschuehly.htmx.spring.supabase.auth.SupabaseAutoConfiguration$$SpringCGLIB$$FastClass$$1.invoke(<generated>) ~[htmx-supabase-spring-boot-starter-0.3.0.jar:na]
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:258) ~[spring-core-6.1.3.jar:6.1.3]
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:331) ~[spring-context-6.1.3.jar:6.1.3]
	at de.tschuehly.htmx.spring.supabase.auth.SupabaseAutoConfiguration$$SpringCGLIB$$0.supabaseClient(<generated>) ~[htmx-supabase-spring-boot-starter-0.3.0.jar:na]
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103) ~[na:na]
	at java.base/java.lang.reflect.Method.invoke(Method.java:580) ~[na:na]
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:140) ~[spring-beans-6.1.3.jar:6.1.3]
	... 48 common frames omitted
```

this error throw `SupabaseClientBuilder` 

```kt
 init {
        val module = when {
            supabaseUrl.contains("realtime/v1") -> "realtime/v1"
            supabaseUrl.contains("auth/v1") -> "auth/v1"
            supabaseUrl.contains("storage/v1") -> "storage/v1"
            supabaseUrl.contains("rest/v1") -> "rest/v1"
            else -> null
        }
        if(!ignoreModulesInUrl && module != null) { // this is the problem
            error("The supabase url should not contain ($module), supabase-kt handles the url endpoints. If you want to use a custom url for a module, specify it within their builder but that's not necessary for normal supabase projects")
        }
        if(supabaseUrl.startsWith("http://")) {
            useHTTPS = false
            Logger.w("Core") { "You are using a non HTTPS supabase url ($supabaseUrl)."}
        }
    }
```
but `SupabaseAutoConfiguration` supabaseUrl has '/auth/v1'
look below

```kt
    @Bean
    @ConditionalOnMissingBean
    fun supabaseClient(supabaseProperties: SupabaseProperties): Auth {
        val supabase = createSupabaseClient(
            supabaseUrl = "https://${supabaseProperties.projectId}.supabase.co/auth/v1",
            supabaseKey = supabaseProperties.anonKey
        ) {
            install(Auth) {
                autoSaveToStorage = false
                autoLoadFromStorage = false
                alwaysAutoRefresh = false
            }
        }
        return supabase.auth
    }
```

so my project doesn't start anyway

how can i solve this problem and why you didn't see this error?
i think my project configure must be wrong could you let me know?

**Comment by tschuehly** [comment]
Hey @yevgnenll 

this issue is fixed in 0.3.2. Update the dependency and let me know if it fixes the problem.

Best 
Thomas

**Comment by yevgnenll** [comment]
thx fixed it!

i think why don't you modify README?
